pkg/printers/teamcity.go

```diff
@@ -100,7 +100,7 @@ type InspectionInstance struct {
 	message             string // (optional)  limited by 4000 characters.
 	file                string // (mandatory) file path limited by 4000 characters.
 	line                int    // (optional) line of the file, integer.
-	additionalAttribute string
+	additionalAttribute string // (optional) severity of the inspection; debug, info, warning, error, etc.
```

Comment:
```suggestion
	additionalAttribute string
```

I only added comments from the spec.
This comment is not in the same way, I think it's better to not have it.

Reply:
I thought about completeness but your suggestion sounds good to me.